Detectors/MUON/MCH/Raw/Common/include/MCHRawCommon/SampaCluster.h

```diff
@@ -43,15 +43,15 @@ struct SampaCluster {
   ///
   /// if some parameter does not fit within its expected range
   /// a std::invalid_argument exception is thrown.
-  explicit SampaCluster(uint16_t timestamp, uint32_t chargeSum);
+  explicit SampaCluster(uint16_t timestamp, uint32_t bunchCrossing, uint32_t chargeSum);
 
   /// Constructs a cluster which holds a vector of raw samples
   /// \param timestamp must fit within 10 bits
   /// \param samples : each sample must fit within 10 bits
   ///
   /// if some parameter does not fit within its expected range
   /// a std::invalid_argument exception is thrown.
-  SampaCluster(uint16_t timestamp, const std::vector<uint16_t>& samples);
+  SampaCluster(uint16_t timestamp, uint32_t bunchCrossing, const std::vector<uint16_t>& samples);
```

Comment:
also for clarity I was thinking to use uint10_t and uint20_t typedefs to make the API even clearer. What do you think ?

Reply:
Good idea, I introduced the two types in the SampaCluster structure, via a "using" keyword.